mat_nn_hip.hpp

```diff
@@ -101,19 +105,25 @@ double su3_mat_nn(std::vector<site> &a, std::vector<su3_matrix> &b, std::vector<
 
   // benchmark loop
   auto tstart = Clock::now();
+  tprofiling = tstart;
```

Comment:
Is there any reason we need two separate variables to hold the start time? They seem to be doing the same thing in this loop.